src/dgrid/DgridWrapper.ts

```diff
@@ -0,0 +1,43 @@
+import { DNode } from '@dojo/framework/widget-core/interfaces';
+import { WidgetBase } from '@dojo/framework/widget-core/WidgetBase';
+import { diffProperty } from '@dojo/framework/widget-core/decorators/diffProperty';
+import { DgridWrapperProperties, constructionKeys } from './DgridWrapperProperties';
+import { DgridInnerWrapper, DgridState } from './DgridInnerWrapper';
+import { w } from '@dojo/framework/widget-core/d';
+import { columnsDiff } from './diff';
+
+const keyPrefix = 'dgridWrapper';
+
+@diffProperty('columns', columnsDiff)
+export class DgridWrapper extends WidgetBase<DgridWrapperProperties> {
+	key = 0;
+	gridState: DgridState;
```

Comment:
should add the visibility of this property... I assume `private` again. If so then, we prefix privates with `_`